src/main/io/osd.c

```diff
@@ -1146,6 +1146,53 @@ static bool osdDrawSingleElement(uint8_t item)
     return true;
 }
 
+#ifdef USE_OSD_STICK_OVERLAY
+static void osdDrawStockOverlayAxis(uint8_t xpos, uint8_t ypos)
+{
+
+    for (uint8_t x = 0; x < OSD_STICK_OVERLAY_WIDTH; x++) {
```

Comment:
`unsigned x = 0`

Reply:
What do you mean here?

Reply:
Using non-native types on ARM si suboptimal (gcc emits uint8_t -> unsigned on almost every access).
(my opinion, not generally shared between BF programmers): You want unsigned variable, but size is not important. Using `uint8_t` looks like you want to limit variable size for some reason. Using `uint32_t` (which is equivalent to `unsigned` on target processor) looks like there is some reason to use exactly 32 bits. It is confusing when reviewing code (you have to think about it a little bit).
I prefer to stick to `signed` (sign importance is stressed), `int` - just a number and `unsigned` - unsigned number is enough (usually to avoid signed/unsigned warning with `sizeof` etc.)

Reply:
Interesting, that's correct.  Just unsigned by itself is not very portable, but I guess the days of 16-bit processors are over.  I'll change it to unsigned x.  The same goes for void osdDrawStockOverlayAxis(uint8_t xpos, uint8_t ypos), to void osdDrawStockOverlayAxis(unsigned xpos, unsigned ypos) then?